docs/source/installation/windows.rst

```diff
@@ -0,0 +1,43 @@
+********************
+Installation on Windows
+********************
+
+Python package
+==============
+
+1. `Install Python <https://www.python.org/downloads>`_ version 2.7.12, 2.8 or 3.4 upwards - Python 2.7.12 onwards, contains pip in the installation package.
+
+2. Install Powerline using one of the following commands:
+
+   .. code-block:: ps1
+
+       pip install --user powerline-status
+
+   will get current release version.
+
+   .. note::
+      Powerline developers should be aware that ``pip install --editable`` does 
+      not currently fully work. Installation performed this way are missing 
+      ``powerline`` executable that needs to be symlinked. It will be located in 
+      ``scripts/powerline``.
+
+Vim installation
+================
+
+Before you begin the installation, ensure that your Vim and Python supported CPU architecture are either both 32bit or both 64bit. Python support in Vim will not work if they are different.
```

Comment:
Lines are too long. Documentation must wrap at 80’th column unless it is a code block.